packer/linux/scripts/install-docker.sh

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 DOCKER_COMPOSE_V2_VERSION=2.27.0
-DOCKER_BUILDX_VERSION=0.14.0
```

Comment:
i don't think [v0.15 exists yet...](https://github.com/moby/buildkit/releases/tag/v0.15.0)

Reply:
My link was a bit misleading, sorry! [buildx v0.15.0](https://github.com/docker/buildx/releases/tag/v0.15.0) has been released, which contains [buildkit v0.14.0](https://github.com/moby/buildkit/releases/tag/v0.14.0), which has the bugfix.

Reply:
ah of course! my bad. on another note, the dependencies section in the moby/docker changelogs is quite nice, i wonder if we could steal that for the  agent....